src/FixedPointNumbers.jl

```diff
@@ -8,6 +8,8 @@ import Base: ==, <, <=, -, +, *, /, ~, isapprox,
              div, fld, rem, mod, mod1, fld1, min, max, minmax,
              rand, length
 
+import Statistics: _mean_promote
```

Comment:
I don't think it is "free". Time is money.:stuck_out_tongue_closed_eyes:

https://github.com/JuliaLang/Statistics.jl/pull/25
The commit was merged on Apr. 14, 2020.

Reply:
> Time is money

Development time or load time? Statistics is built into Julia itself, so I think it doesn't cost anything to import it here (any tiny amount of time is spent during precompilation). That's the sense of "free" I meant.

Reply:
I meant we have to wait a lot of time until `_mean_promote` is available in all the supported versions (i.e. until the release of v1.0.6 or the next LTS). Of course, this is a joke. :smile:

Reply:
Very good. Nice catch, it's much better to make it conditional, see if this version is acceptable.